src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,85 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Calculator {
+
+    private final String expression;
+
+    public Calculator(String expression){
+        validInput(expression);
+        this.expression = expression;
+    }
+
+    public String getExpression() {
+        return expression;
+    }
+
+    public int calculate() {
+
+        String[] splitExpression = splitInput(expression);
+        validInteger(splitExpression[0]);
+
+        AtomicInteger result = new AtomicInteger(Integer.parseInt(splitExpression[0]));
+        List<Integer> valueList = getValue(splitExpression);
+        List<String> operatorList = getOperator(splitExpression);
+
+        for(int i=0;i<valueList.size()-1;i++){
+            result.set ( cal(result.get(), valueList.get(i+1), operatorList.get(i)) );
+            System.out.println("==>"+result.get());
+        }
+
+        return result.get();
+    }
+
+    // 유효성 검사
+    public void validInput(String input) {
```

Comment:
```suggestion
    private void validInput(String input) {
```
private 접근 제어자를 사용해 외부 클래스로부터 접근을 막아보세요 :) 
대부분의 메소드 접근제어자를 public 으로 오픈시켜두었는데 외부클래스로 제공해야하는 인터페이스가 아니면 private / protected 접근 제어자를 사용해보는건 어떨까요 ?